recipes/traitlets/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "4.2.1" %}
+
+package:
+  name: traitlets
+  version: {{ version }}
+
+source:
+  fn: traitlets-{{ version }}.tar.gz
+  url: https://github.com/ipython/traitlets/archive/{{ version }}.tar.gz
+  md5: fc7f46a76b99ebc5068f99033d268dcf
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - ipython_genutils
+    - decorator
+    # enum34 may be of interest here also, but a test would also
+    # need to be added to the recipe. 
```

Comment:
> Should we just add the optional dependency?

I couldn't quickly write a test which needed it so didn't add the optional dependency. As a first cut, I think this is good enough without.